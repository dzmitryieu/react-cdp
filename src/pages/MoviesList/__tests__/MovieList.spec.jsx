import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import MovieList from '../index';

const shallow = new ShallowRenderer();

const onClick = jest.fn();

describe('MovieInfo Snapshot', () => {
  test('render', () => {
    const component = shallow.render(
      <MovieList onClick={onClick} />,
    );
    expect(component).toMatchSnapshot();
  });
});
