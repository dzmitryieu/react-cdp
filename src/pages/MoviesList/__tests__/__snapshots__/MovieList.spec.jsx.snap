// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieInfo Snapshot render 1`] = `
<div
  className="wrapper"
>
  <section
    className="section"
  >
    <MovieInfo
      films={
        Array [
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "13456",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "3636",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "547568",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "25567",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "78909786",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "546575",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "78908484",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "5365878",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
          Object {
            "genres": Array [
              "Drama",
              "Comedy",
            ],
            "id": "9837609",
            "name": "Captain Marvel",
            "poster_path": "../../assets/images/poster.jpg",
            "year": "1984",
          },
        ]
      }
      onClick={[MockFunction]}
    />
  </section>
</div>
`;
